Tests/SwiftDocCTests/Rendering/RenderNodeTranslatorTests.swift

```diff
@@ -1154,7 +1154,8 @@ class RenderNodeTranslatorTests: XCTestCase {
          let roundTrippedArticle = try JSONDecoder().decode(RenderNode.self, from: encodedArticle)
     
          XCTAssertEqual(roundTrippedArticle.icon?.identifier, "plus.svg")
-    
+
+         XCTAssertEqual(renderNode.metadata.customMetadata.count, 1)
          XCTAssertEqual(
```

Comment:
@ethan-kusters  i see customMetadata present in the renderNode but not when i get the data from encodedArticle. Is there something i am missing? does it depend on that any other frontend changes?

Reply:
I think we're just missing the encoding of the data. I left a comment here: https://github.com/apple/swift-docc/pull/405#discussion_r999792740.

Reply:
ah! Correct, i added that.